python/learn.py

```diff
@@ -71,7 +73,11 @@ def get_progress():
     lesson = int(options['--lesson'])
     fast_simulation = not bool(options['--slow'])
 
-    env = UnityEnvironment(file_name=env_name, worker_id=worker_id, curriculum=curriculum_file)
+    if seed != -1:
```

Comment:
Can you explain a bit why this check exists? Why is the default pseudo randomizer explicitly to -1?